packages/iTwinUI-react/src/core/Table/actionHandlers/selectHandler.ts

```diff
@@ -56,6 +56,109 @@ export const onSelectHandler = <T extends Record<string, unknown>>(
   onSelect?.(selectedData, newState);
 };
 
+/**
+ * Handles selection when clicked on a row while shift is selected.
+ */
+export const onShiftSelectHandler = <T extends Record<string, unknown>>(
+  state: TableState<T>,
+  action: ActionType,
+  instance: TableInstance<T>,
+) => {
+  const resetSelection = () => {
+    instance.toggleAllRowsSelected(false);
+  };
+
+  const getGreaterId = (id: string) => {
+    const greaterId = id.split('.');
```

Comment:
Ohh, I'm wondering, can users pass their own id? I thought the React Table package automatically gives the ids to the rows and they are unchangable. Is that true? If it is, then maybe we won't allow users to pass a custom `getGreaterId()` since they can't pass custom ids. Also, to confirm, what is GUID? Is it "greater ID"?